source/global.cc

```diff
@@ -23,6 +23,39 @@
 
 namespace aspect
 {
-  const double year_in_seconds  = 60*60*24*365.2425;
```

Comment:
You can not simply remove year_in_seconds because it is used in several places in the code.